docs/deprecations.md

```diff
@@ -149,3 +149,47 @@ Please use `onUnauthenticatedRequest` instead. `allowUnauthenticated` will be re
 ## `bin/serverless`
 
 Please use `bin/serverless.js` instead. `bin/serverless` will be removed with v2.0.0
+
+## To Disable specific Deprecations
+
+To disable specific deprecations set `SLS_DEPRECATION_DISABLE` environment variable.
```

Comment:
I would write this whole section in one paragraph, without putting it on new lines, so something like this:

```
To disable specific deprecations set `SLS_DEPRECATION_DISABLE` environment variable. Setting `SLS_DEPRECATION_DISABLE=*` will disable all deprecations. If you want to disable specific depreciations set `SLS_DEPRECATION_DISABLE=CODE1,CODE2`.
```